### ISSUE TITLE
Uncaught RangeError: Source is too large at Uint8Array.set (native) .........

### ISSUE THREAD

**Issue by Prahadeesh92** [bpampuch/pdfmake#1113]
Error: Uncaught RangeError: Source is too large
    at Uint8Array.set (native)...................................

I got the above error with my previous version of pdfmake which was "0.1.20".  I updated pdfmake to the latest version "0.1.32" and my Chrome version is "60.0.3112.101". I don't have Adblock extension in chrome. Still, when I am trying to print, I am getting the same error "Uncaught RangeError: Source is too large". Is there any solution to solve this issue?

**Comment by kmjones77** [comment]
I also get this issue. I do have adblock, Chrome version 60.0.3112.101.

**Comment by kmjones77** [comment]
Are you sure you have cleared cache? I cleared my cache in Chrome and this issue no longer occurs.

**Comment by Prahadeesh92** [comment]
Yeah, I tried after clearing my browser's cache but the issue still exists.

**Comment by JohhnyP** [comment]
Reported in here also
https://github.com/bpampuch/pdfmake/issues/998

**Comment by liborm85** [comment]
This problem is in 0.1.32 solved. Try to check which version you are using.
Eventually attach example for reproduce your problem.

**Comment by staminna** [comment]
0.1.32 didn't fix it for me, therefore I updated to 0.1.33 and the issue remained the same.
Here is how I declare the docDefinition in my controller.js file:

```
html2canvas(document.getElementsByClassName('content'), {
  onrendered: function (canvas) {
    var data = canvas.toDataURL();
    var docDefinition = {
    content: [{
	image: data,
	width: 500,
    }]
  };
pdfMake.createPdf(docDefinition).print();
}
```
I tried the latest versions of html2canvas and also directly providing images from URL instead of 'data' from the canvas but then the error is:


`image: 'data:image/jpeg;base64' + 'https://s.imgur.com/images/logo-1200-630.jpg',`
> Uncaught (in promise) TypeError: Cannot read property 'embed' of undefined.

I must be wrong but its maybe a memory or resolution issue, no?
